index.js

```diff
@@ -116,7 +114,14 @@ module.exports = (input, opts) => {
 		},
 		{
 			title: 'Publishing package',
-			task: () => exec('npm', ['publish'].concat(opts.tag ? ['--tag', opts.tag] : []))
+			task: () => readPkgUp()
```

Comment:
I would use `readPkgUp.sync()` here. No real benefit of using async `fs` here as these tasks are done serially.